async-nats/CHANGELOG.md

```diff
@@ -4,7 +4,6 @@ This release focuses on fixes and improvements, with addition of ordered Push Co
 
 ## Added
 * Add `get_last_raw_message_by_subject` to `Stream` by @caspervonb in https://github.com/nats-io/nats.rs/pull/584
-* Add cluster to streaminfo/consumerinfo struct by @j13tw in https://github.com/nats-io/nats.rs/pull/537
```

Comment:
Huh? why the removal?

Reply:
this is the change for sync client, not async. A minor thing, but cleans it up.